Q/Qiskit/build_tarballs.jl

```diff
@@ -0,0 +1,57 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Qiskit"
+version = v"2.2.3"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Qiskit/qiskit.git", "8f33595f5b9e9c99b7aa81002655d13f48c8ac1b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+export PYTHONHOME=${prefix}
+cd qiskit
+mkdir -p target
+export PYO3_PYTHON=/workspace/destdir/bin/python3
+export RUSTFLAGS="-L ${libdir}"
+
+# avoid 'cannot create cdylib' error on musl targets
+# see https://github.com/rust-lang/cargo/issues/8607
+#     https://github.com/rust-lang/rust/issues/59302
+if [[ "${target}" == *-musl* ]]; then
+    export RUSTFLAGS="${RUSTFLAGS} -C target-feature=-crt-static"
+fi
+
+make C_CARGO_TARGET_DIR=target/${rust_target}/release -j${nproc} c
+install -Dvm 755 "dist/c/lib/libqiskit.${dlext}" "${libdir}/libqiskit.${dlext}"
+cp -vr dist/c/include/* "${includedir}"
+"""
```

Comment:
```suggestion
install_license LICENSE.txt
"""
```
Nice to be explicit about this.